storybook/src/stories/Filters/molecules/SortDropdown.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import styled from 'styled-components';
+import { IFilterItem, SortDropdown } from 'scorer-ui-kit';
+import { boolean, object, select } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Filters/molecules',
+  component: SortDropdown,
+  decorators: [],
+  parameters: {
+    jsx: { skip: 1 }
+  }
+};
+
+const Container = styled.div``;
+
+export const _SortDropdown = () => {
+  const language = select("Language", { English: 'english', Japanese: "japanese" }, "japanese");
+
+  const dropdownList: IFilterItem[] = useMemo(() => [
+    { text: language === 'english' ? 'Name' : '名前', value: 'name' },
+    { text: language === 'english' ? 'Status' : 'ステータス', value: 'status' },
+    { text: language === 'english' ? 'Cost' : '価格', value: 'cost' }
+  ], [language])
+
+  const [sortSelected, setSortSelected] = useState<IFilterItem>({ text: dropdownList[1].text, value: dropdownList[0].value });
+  const [isAscending, setIsAScending] = useState<boolean>(true);
+  const isLoading = boolean('Is Loading', false);
+
+  // const buttonText = text('Button Text', `Sorted by ${dropdownList[0].text}`);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const list = object('Dropdown list', dropdownList)
```

Comment:
I can't see where this line is being used. Can you remember why you have included it but disabled the linting on it?

Reply:
It's just for reference of how the object should look like, but it's not meant to be able to edit it.